### ISSUE TITLE
ReferenceError: Internationalization is not defined

### ISSUE THREAD

**Issue by benzBrake** [theroyalstudent/layered#32]
I try to install layered to transmission on debian 7. But it do not work.
transmission version
```
root@sb2:~# transmission-cli -v
transmission-cli 2.52 (13304)\
```
Error Log
```
Uncaught ReferenceError: Internationalization is not defined
    at app.min.js:13
    at dispatch (app.min.js:2)
    at v.handle (app.min.js:2)
```
Browser: Firefox 51

**Comment by xF4m3** [comment]
Same in Chrome Version 58.0.3029.110 (64-bit) and transmission-cli 2.92.
Install an older version, grab one from the realease page thats not the latest. 

Basically the website is totally unusable. You get the normal page but clicking doesnt really do anything at all, it also doesnt show any torrents and so in.

**Comment by runbiscuit** [comment]
Hi all, this has been fixed in a new commit in the `master` branch. Let me know how it is.

**Comment by xF4m3** [comment]
Yeah its working again, thanks :), also a qquick question, could i maybe upload a diffrent style.css and you could maybe add an option to switch between them?

**Comment by runbiscuit** [comment]
@xF4m3: Are you intending to do really big changes to the interface? If not, I would suggest adding your own CSS by adding another file named `user.css` in `/resources/css/`, and adding a `<link href="...">` line in the index.html file.

I'm currently not looking to adding too many options for users, as it may get confusing.
**_However, I can add the `<link href="...">` line for `index.html` in the next commit._**

Cheers! :)

**Comment by xF4m3** [comment]
Ohh okay, thanks.
No i just have some color changes in there, aswell as a 2 rows from top to bottom with the torrents.
The user.css should be enough for me then. 
I jsut realised that today, but could you maybe add the option to add trackers to an already existing torrent? Aswell as an option to recheck the downlaoded torrent files in case you moved them over from somewhere else? Atm you need to remove the torrent from the list and add it again so it reckecks the files on the harddrive. If yes, i would open a new issue aobut that. 

Also, as always thanks for your work on this :+1:
